src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/ShowLocationHistory/readme.md

```diff
@@ -0,0 +1,41 @@
+# Show location history
+
+Display your location history on the map.
+
+![](ShowLocationHistory.jpg)
+
+## Use case
+
+You can track the user's location and display that on a map, for example to visualize how the user has moved through the world or to allow the user to re-trace their steps.
+
+## How to use the sample
+
+Click, 'Start tracking' to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Click 'Stop tracking' to stop updating the location history.
+
+## How it works
+
+1. If necessary, the app requests location permissions from the operating system.
```

Comment:
Should this simply be "If necessary, request location ..." instead of having "the app" do the action? It would match better with the following steps.